packages/next-on-pages/docs/worker-invocation.md

```diff
@@ -0,0 +1,18 @@
+# Opt-Out Assets from Invoking the Worker
```

Comment:
I am not really convinced about this title and the file name, as per the file name what about something like `static-assets-routing.md`? or something along those lines?

The title could also be `# Static assets routing` or simply `# _routes.json` 🤔